platform-includes/distributed-tracing/how-to-use/javascript.astro.mdx

```diff
@@ -2,6 +2,14 @@ If you're using our Astro SDK, distributed tracing will work out of the box for
 
 When you are interacting with other external API systems, you might have to define `tracePropagationTargets` to get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues.
 
+<Alert>
+
+Note: port numbers are relevant for trace propagation and the origin. You may need to configure the `tracePropagationTargets` to ensure that traces are propagated across your services if they run on different ports. 
+
+For example, if you have a Vite frontend running on `http://localhost:5173` and a Bun backend running on `http://localhost:3000`, both `http://localhost:5173` and `http://localhost:3000` should be added to the `tracePropagationTargets` array.
```

Comment:
Hmm this is not correct I believe, you do not need to add your frontend as `traceProapgationTargets` as you'll usually not make API requests to it?

Reply:
Yup - you're absolutely correct, I have a fix pushed up for this. I kept with the previous examples that had localhost in them - but its not entirely useful or needed. Thanks for catching.